.travis.yml

```diff
@@ -4,3 +4,4 @@ sudo: false
 
 rvm:
   - 2.2
+  - 2.3
```

Comment:
Just had Travis bork with `2.3` on Global ID the other day, predict we need `2.3.0`.